### PR TITLE
feat(TT-3): user authentication tests

### PR DESCRIPTION
**Pull request by flyindonut**
**JIRA:** 🌸 [TT-3](https://champlainsaintlambert.atlassian.net/browse/TT-3) 🌸

## Context:

✨ Added tests + DatabaseLoader for Users ✨

### backend:
🔧 Added Tests + DatabaseLoader for Users + Minor Refactoring ✨

### front end:
🌸E2E Tests 🌈

### CODE REVIEW

File: be-traveltrove/src/main/java/com/traveltrove/betraveltrove/business/traveler/TravelerService.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package com.traveltrove.betraveltrove.business.traveler;
 
-import com.traveltrove.betraveltrove.presentation.travaler.TravelerRequestModel;
-import com.traveltrove.betraveltrove.presentation.travaler.TravelerResponseModel;
```
Comment (CharlesS42) [inline be-traveltrove/src/main/java/com/traveltrove/betraveltrove/business/traveler/TravelerService.java:4]: Oh my god! Well done spotting the 1 letter difference from 'Travaler' to 'Traveler' !

File: be-traveltrove/src/main/java/com/traveltrove/betraveltrove/presentation/traveler/TravelerController.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.traveltrove.betraveltrove.presentation.travaler;
+package com.traveltrove.betraveltrove.presentation.traveler;
```
Comment (ioannisp03) [inline be-traveltrove/src/main/java/com/traveltrove/betraveltrove/presentation/traveler/TravelerController.java:1]: was wondering how you had 19 files changed just for tests 😆

**Review by ioannisp03 — APPROVED** [review]
looks good

**Review by CharlesS42 — APPROVED** [review]
Overall, tests look good. I see sample data as well that meet the criteria.

**Pull request merged**